flatten.py

```diff
@@ -31,5 +31,5 @@ def isAtomicType(thing):
 
 def run(objectToFlatten):
     flattenedObject = collections.OrderedDict(); # maintains insertion order, which looks neater
-    flatten(unflattenedJsonObject, "", flattenedObject) # initialize the flatten() with an empty string key prefix
+    flatten(objectToFlatten, "", flattenedObject) # initialize the flatten() with an empty string key prefix
```

Comment:
In ingest-archiver we used an existing python module to flatten the JSON. Please consider using this in place of the custom flatten code specifically: flatten_json